R/factor2string.R

```diff
@@ -22,33 +22,33 @@
 #'
 #' @examples
 #' library(modeldata)
-#' data(okc)
+#' data(Sacramento)
 #'
-#' rec <- recipe(~ diet + location, data = okc)
+#' rec <- recipe(~ city + zip, data = Sacramento)
 #'
 #' rec <- rec %>%
-#'   step_string2factor(diet)
+#'   step_string2factor(city)
```

Comment:
The Sacremento data set isn't good for this step since `city` and `zip` are already factor variables.

I think this overall example is kinda weird and should mirror the examples for `step_string2factor()`.